### PR TITLE
Add configuration documentation for the Backup component

### PR DESCRIPTION
**Pull request by kazydek**
Changes proposed in this pull request:

- Created configuration docs for this Backup chart:
   - [`velero`](https://github.com/kyma-project/kyma/tree/master/resources/velero)

**Related issue(s)**
See also #4053

### CODE REVIEW

**Comment by kazydek** [comment]
@a-thaler could you:
- let me know if that generic list and links to the official documentation with configurable parameters are ok for you?
- test these configuration, as discussed last week?

**Comment by kazydek** [comment]
@a-thaler I am also a bit confused as to the info in the [`README.md`](https://github.com/kyma-project/kyma/blob/master/tests/end-to-end/backup-restore-test/README.md#run-end-to-end-tests-locally) doc for the Backup and Restore tests. It is saying that:
```
NOTE: Before running the test, configure Velero using these guidelines.
```

It is pointing to the [configuration do](https://kyma-project.io/docs/1.2/components/backup/#details-configuration) describing how to create the two CRs that we decided to remove as the new configuration do will cover this area. The question is whether the configuration is in fact required as mentioned in the Note panel? Should that be done through creating custom resources or overrides (ConfigMaps, Secrets)? Could you shed a bit light on what to do with the Note panel and whether in fact the doc to which it points should be removed? Thanks.

File: docs/backup/05-01-velero.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.volumeSnapshotLocation** | The location to store volume snapshots created for the backup. It is cloud provider-specific. [See](https://velero.io/docs/v1.0.0/api-types/volumesnapshotlocation/) the official Velero documentation for the full list of configurable parameters. | None |
```
Comment (a-thaler) [inline docs/backup/05-01-velero.md]: It is hard to understand how I should pass the data from the documentation tot his attribute. Can you improve it by either give an example or make clear that the content of the "spec" attribute of a volumesnapshotlocation resource needs to be set to the property

File: docs/backup/05-01-velero.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* [Top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides)
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **global.volumeSnapshotLocation** | The location to store volume snapshots created for the backup. It is cloud provider-specific. For example, to set up `aws` as the provider of the snapshot storage location, override the value for the **global.volumeSnapshotLocation.spec.provider** parameter. [See](https://velero.io/docs/v1.0.0/api-types/volumesnapshotlocation/) the official Velero documentation for the full list of configurable parameters and examples. | None |
```
Comment (a-thaler) [inline docs/backup/05-01-velero.md]: I somehow was wrong, sorry for that, it is not that complicated at all.

 global.volumeSnapshotLocation.name = name of the cloud provider
 global.volumeSnapshotLocation.bucket = name of the bucket to use in the object store for the backups

That are the supported config attributes
global.volumeSnapshotLocation.config.region
global.volumeSnapshotLocation.config. apitimeout

File: docs/backup/05-01-velero.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+| **global.backupStorageLocation** |  The location to store backups. It is cloud provider-specific. For example, to set up `aws` as the provider of the backup storage location, override the value for the **global.backupStorageLocation.spec.provider** parameter. [See](https://velero.io/docs/v1.0.0/api-types/backupstoragelocation/) the official Velero documentation for the full list of configurable parameters and examples. | None |
```
Comment (a-thaler) [inline docs/backup/05-01-velero.md]: Same simplification here: In general here is the docu: https://github.com/heptio/velero/blob/release-0.11/docs/api-types/backupstoragelocation.md

global.backupStorageLocation.name = cloud provider name
global.backupStorageLocation.bucket = name of the bucket to use in the object store for the backups
global.backupStorageLocation.prefix = The directory inside a storage bucket where backups are to be located

That are the supported attributes like described in docu
global.backupStorageLocation.config.region
global.backupStorageLocation.config. s3ForcePathStyle
global.backupStorageLocation.config. s3Url
global.backupStorageLocation.config. kmsKeyId
global.backupStorageLocation.config. resourceGroup
global.backupStorageLocation.config. storageAccount
global.backupStorageLocation.config. publicUrl

**Pull request merged**